src/Randomizer.SMZ3.Tracking/tracker.json

```diff
@@ -1541,6 +1660,8 @@
     "ActionUndone": [
       "Was that not what you wanted?",
       "Make up your mind.",
+      "I'll undo that",
+      "My mistake",
```

Comment:
This is honestly pretty fair, although I'm not sure if this is quite in the 'spirit' of tracker. Maybe make it a little rarer?